src/turbolinks/controller.coffee

```diff
@@ -87,10 +87,14 @@ class Turbolinks.Controller
   getCachedSnapshotForLocation: (location) ->
     @cache.get(location)
 
+  shouldCacheCacheable: ->
```

Comment:
:dizzy_face: 

Fixing it now, and I'm also organizing better the PR commits (my last push introduced unnecessary back-and-forth in the code changes)